portal/index.html

```diff
@@ -0,0 +1,59 @@
+<html>
+
```

Comment:
Missing <head>, <title> and <body> ?
Maybe also add utf-8 and no referrer meta tag (not sure for the no referer)

Reply:
This was just a yolo page to test the new API, the actual new portal is being worked on in https://github.com/YunoHost/yunohost-portal

Reply:
sorry I'm out of line. Thanks for the info and replying.